src/Form/BecomeAdherentType.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class BecomeAdherentType extends AbstractType
```

Comment:
A membership is about becoming adherent. Please keep this one as `MembershipRequestType`.

Reply:
Ok

Reply:
Just did it and I don't like it. It's more confusing IMO.
I understand your point but from an external point of view this name makes more sense.
It is really obvious to understand and it's the main goal here no ?
You can become an adherent with this form, that's it.
What do you think ?

Reply:
`AdherentType` <3. Thanks!